Libraries/Hotcakes.PaypalWebServices/Hotcakes.PaypalWebServices.csproj

```diff
@@ -65,15 +65,12 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="PayPalCheckoutSdk, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\PayPalCheckoutSdk.1.0.4\lib\netstandard2.0\PayPalCheckoutSdk.dll</HintPath>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\References\Misc\PayPalCheckoutSdk.dll</HintPath>
```

Comment:
Is this not available using Nuget, or is there another issue with trying that?

Reply:
Okay. Is this update something that we can see compiled in this source code, or is it somewhere else?  We need to be able to maintain the fork.  

Also, is this update something that should be updated in the original source code for the DLL?  If yes, there should be a PR there too, so we don't accidentally lose this update at some point in the future.

Reply:
The original Repository is no longer maintained since a couple of months, that is why I didn't make a PR to it. I think the best solution could be create our own repository and push my updated code there.

Reply:
So, do we have any code to maintain outside of Hotcakes to maintain this, or is the Nuget import all that we need now?